superset/views/core.py

```diff
@@ -2001,13 +2001,13 @@ def dashboard_permalink(  # pylint: disable=no-self-use
             return redirect("/dashboard/list/")
         if not value:
             return json_error_response(_("permalink state not found"), status=404)
-        dashboard_id = value["dashboardId"]
+        dashboard_id, state = value["dashboardId"], value.get("state", {})
```

Comment:
is this standard styling? i would just use 2 lines tbh

Reply:
Nah, just my style choice. Since they are both derived from the same variable. It's more concise to be written in one line. This is similar to object expansion in JS.